common/src/main/resources/db/changelog/test_only/coverage-fake-partitions.sql

```diff
@@ -19,4 +19,10 @@ create table if not exists tst_2022 partition of tst
     for values in (2022);
 
 create table if not exists tst_2023 partition of tst
-    for values in (2023);
\ No newline at end of file
+    for values in (2023);
+
+create table if not exists tst_2023 partition of tst
```

Comment:
Should the create statement here and on line 27 be checking against the upcoming year instead of `tst_2023`?

Ex:
`create table if not exists tst_2024 partition of tst`
`create table if not exists tst_2024 partition of tst`

Reply:
oops! I mistyped
Fixed